src/image-copy/azext_imagecopy/custom.py

```diff
@@ -127,13 +153,11 @@ def create_resource_group(resource_group_name, location):
 
     cmd_output = run_cli_command(cli_cmd)
 
-    if 'true' in cmd_output:
-        return
-
-    # create the target resource group
-    logger.warn("Creating resource group: %s", resource_group_name)
-    cli_cmd = prepare_cli_command(['group', 'create',
-                                   '--name', resource_group_name,
-                                   '--location', location])
+    if 'false' in cmd_output:
```

Comment:
Just being curios - why did you change this?

Reply:
That's my mistake. I applied a patch, based on older version of the script. So it's a regression I would say. I'll revert it back